docs/openqaoa-jobs/braket-jobs.md

```diff
@@ -0,0 +1,28 @@
+# Bryong your own OpenQAOA container to Amazon Braket Hybrid Jobs
+
+If you are wondering what is an Amazon Braket Hybrid Jobs, check out Braket's very answer
+
+> With Hybrid Jobs, you can run algorithms that use both classical and quantum compute resources. These hybrid quantum-classical algorithms can help you optimize the performance of the quantum devices currently available. These algorithms are designed to maximize the benefits and reduce the drawbacks of both types of computational systems and are generally used to find the ground state or global minimum of a particular system.
+
+QAOA is a great example of such hybrid quantum-classical algorithms, and we here provide a simple procedure to prepare your own OpenQAOA container.
+
+## Steps to create your own OpenQAOA container
+
+First of all, these instructions are based on Amazon Braket's documentation. Therefore, a safe place to check if you are an experienced Docker and AWS practitioner is to head to the [Bring your own container](https://docs.aws.amazon.com/braket/latest/developerguide/braket-jobs-byoc.html) page!
+
+The following instructions will allow you to create your OpenQAOA container. Note that they will work out of the box only for Linux and macOs.
+ 
+ 1. Make sure all the dependencies are installed and credentials are properly set:
+	 - Clone openqaoa from github `git clone https://github.com/entropicalabs/openqaoa.git`
```

Comment:
One question to think about -- we should support installing only `openqaoa-braket` to make the docker container work instead of cloning and installing whole of openqaoa. It could be possible if we can make braket jobs work by installing OQ from PyPI apart from cloning too

Reply:
Yes, that is true. We only need to update the docker file though. Maybe I can guide you and Killian through it so that we have some knowledge share?